os/android/drmhwctwo.cpp

```diff
@@ -95,7 +95,7 @@ HWC2::Error DrmHwcTwo::Init() {
   displays_.at(HWC_DISPLAY_PRIMARY).Init(disable_explicit_sync_);
 
   // Start the hwc service
-  hwcService_.Start(*this);
+  // hwcService_.Start(*this);
```

Comment:
Just synced with Kishore, looks like the issue is with latest github code base. The issue was never observed when I had pushed the patch. 
I will sync latest code to reproduce the issue.